migration/migrator/migrations/system/20190816211332_php_ds.py

```diff
@@ -0,0 +1,12 @@
+"""Migration for the Submitty system."""
+import os
+
+
+def up(config):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    """
+    os.system("apt install -qy php-ds")
```

Comment:
I am not familiar with `pecl`, but, after doing a quick search, it looks possible to have different versions by installing using `apt` and `pecl`. 

https://superuser.com/questions/451832/what-is-the-difference-between-apt-get-install-php-apc-pecl-install-apc

To avoid discrepancy between travis and vagrant that may potentially lead to mysterious bugs (although not likely for this simple case), is it possible to all use pecl?

Reply:
In theory, I agree with you that we should use the same, however getting Travis to use apt instead of pecl is a major pain and not really worth the hassle as we transition to BuildBot. On the production server, it's probably better to use apt as it's easier for a sysadmin to manage.